src/components/Register.js

```diff
@@ -16,6 +16,7 @@ function Register() {
     e.preventDefault();
     axios.post(`${BaseUrl}register/`, formData)
       .then((response) => {
+        console.log("-------------check data");
         console.log(response.data);
```

Comment:
Logging full API responses with `console.log` is not recommended, especially if they contain sensitive data or personal information. Suggest to remove or minimize such logs in the production environment to avoid potential security risks.